neptun/src/sleepyinstant/mod.rs

```diff
@@ -52,7 +52,7 @@ impl Instant {
     ///
     /// # Panics
```

Comment:
It seems like a leftover comment. This Panics would also have to be removed.

Reply:
Yup.